pulp_docker/app/serializers.py

```diff
@@ -41,13 +41,14 @@ class ManifestListTagSerializer(platform.ContentSerializer):
     _artifact = MinimalArtifactSerializer(many=False, help_text="File related to this content")
```

Comment:
@dralley: Is the `_artifact` field okay in these serializers?

Reply:
Ah, thank for pointing that out.  It can go, although I don't know what "MinimalArtifactSerializer" is or whether it's necessary to the docker plugin working properly.

Reply:
Contrary to the `MinimalArtifactSerializer` docstring, the `ArtifactSerializer` does include the size.  IMHO, I don't think hiding the digests on the `Artifact` is worth the extra code in these serializers.